src/modules/Packages.rb

```diff
@@ -18,6 +18,10 @@ class PackagesClass < Module
     RESOLVABLE_TYPES = [:product, :patch, :package, :pattern, :language]
     # Minimum set of packages required to enable VNC server
     VNC_BASE_PACKAGES = ["xorg-x11", "xorg-x11-Xvnc", "xorg-x11-fonts", "xinetd"]
+    # Additional packages needed to run second stage in graphical mode
+    # Use exact package name (i.e. libyui-qt6 instead of libyui-qt) to prevent
+    # the UI from asking for confirmation
+    AUTOYAST_X11_PACKAGES = ["libyui-qt6", "yast2-x11"]
```

Comment:
Hardcoding the version to libyui-qt6 would be acceptable for a maintenance branch. For master, it will eventually break.
If there is a popup that requires user interaction, that is indeed bad for AY. Usually such popups can be automatically confirmed by using Report.Message instead of Popup.Message in the code. I also wonder whether this happens with all packages that the user has specified via "Provides".

It seems the better solution would be to explicitly resolve the Provides to package names for the packages that we hardcode. But I wasn't able to find the Pkg binding call for that :-/